profiles/opentelemetry_dev/grafana/dashboards/pulp_insights.json

```diff
@@ -29,15 +35,15 @@
         "x": 0,
         "y": 0
       },
-      "id": 13,
+      "id": 19,
       "panels": [],
-      "title": "Pulp Content",
+      "title": "Disk Usage",
```

Comment:
We need to remove this section here. The disk/space usage metric is not on pulpcore codebase yet.